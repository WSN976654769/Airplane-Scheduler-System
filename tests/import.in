flight import flight.cs
flight import flights.csv
location import locatio.csv
location import locations.csv
flight import flights6.csv
location import locations.csv
location import locatio.csv
flight import fligs.csv
location import locations4.csv
location export locationss.csv
flight export flightss.csv
exit