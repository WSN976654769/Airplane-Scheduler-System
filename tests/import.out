User: Error reading file.

User: Imported 0 flights.
3 lines were invalid.

User: Error reading file.

User: Imported 51 locations.

User: Imported 169 flights.
13 lines were invalid.

User: Imported 0 locations.
51 lines were invalid.

User: Error reading file.

User: Error reading file.

User: Imported 2 locations.

User: Exported 53 locations.

User: Exported 169 flights.

User: Application closed.
